explorer/src/components/SearchBar.tsx

```diff
@@ -265,7 +265,7 @@ function buildTokenOptions(
   if (matchedTokens.length > 0) {
     return {
       label: "Tokens",
-      options: matchedTokens.map(([id, details]) => ({
+      options: matchedTokens.slice(0,10).map(([id, details]) => ({
```

Comment:
also, what do you think about adding this limit @jstarry ? 

The search currently maps hundreds of tokens when a user types in a few letters which makes the search experience feel a bit slow / delayed. Given that the search options revealed to the user are a few at most, I think a limit of appending ~10 tokens at most should be enough.

Reply:
Yeah that's a nice fix, can you make that chance in a separate PR?

Reply:
will do!